### PR TITLE
My solution

### PR DESCRIPTION
**Pull request by kovaliskoveronika**

### CODE REVIEW

File: services/order.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        username: str,
+        date: str = None
+) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        new_order = Order.objects.create(user_id=user.id)
+        if date:
+            new_order.created_at = date
+
+        new_order.save()
```
Comment (Altahoma) [inline services/order.py:19]: `save()` will be calls twice (for the first time during creation of `new_order`) no matter if date provided, so it's better to move `save()` into `if` body.

File: services/user.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+from db.models import User
+from django.contrib.auth import get_user_model
```
Comment (Altahoma) [inline services/user.py:2]: It is worth importing according to PEP8 requirements.

**Review by Altahoma — CHANGES_REQUESTED** [review]

File: services/user.py (hunk 2 above)
Comment (kovaliskoveronika) [reply services/user.py]: I'm not sure that I got your point about it. So can you look at this part again in the new commit?)

File: services/user.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
-from db.models import User
 from django.contrib.auth import get_user_model
 
+from db.models import User
+
```
Comment (Altahoma) [inline services/user.py]: Great job! I meant [imports order](https://peps.python.org/pep-0008/#imports).

**Review by Altahoma — APPROVED** [review]

**Review by Oleg-s-s — APPROVED** [review]
:+1:

**Pull request closed without merging**